apps/api/src/movies/movies.controller.ts

```diff
@@ -20,8 +21,14 @@ export class MoviesController {
    */
 
   @Post()
-  create (@Body() createMovieDto: CreateMovieDto) {
-    return this.moviesService.create(createMovieDto)
+  async create (@Body() createMovieDto: CreateMovieDto) {
+    if (createMovieDto.movieTitles.length === 0) {
+      throw new BadRequestException('At least one movie title is required')
+    }
```

Comment:
You can delegate the validation responsibility to the DTO instead of the Controller

Reply:
Good call! Will remove this